model_args.py

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
+#
+# NVIDIA CORPORATION and its licensors retain all intellectual property
+# and proprietary rights in and to this software, related documentation
```

Comment:
this file is no longer needed, can be removed

Reply:
done.